Crates with build issues: March 2023

- cargo-update 
  - x86_64-sun-solaris: cannot find -lnsl
  - x86_64-unknown-illumos: cannot build openssl due to unused variable
- wasm-pack
  - x86_64-unknown-illumos: cannot build openssl due to unused variable

--- a/crates.json
+++ b/crates.json
@@ -77,7 +77,7 @@
 
     "wasm-pack": {
       "flags": "",
-      "unsupported": "aarch64-pc-windows-msvc,x86_64-sun-solaris",
+      "unsupported": "aarch64-pc-windows-msvc,x86_64-unknown-illumos,x86_64-sun-solaris",
       "git": "https://github.com/rustwasm/wasm-pack",
       "bins": [
         "wasm-pack"
@@ -122,7 +122,7 @@
 
     "cargo-update": {
       "flags": "--features vendored-openssl",
-      "unsupported": "",
+      "unsupported": "x86_64-unknown-illumos,x86_64-sun-solaris",
       "git": "https://github.com/nabijaczleweli/cargo-update",
       "bins": [
         "cargo-install-update",

--- a/crates.json
+++ b/crates.json
@@ -492,7 +492,25 @@
       "bins": [
         "cargo-zigbuild"
       ]
+    },
+
+    "erdtree": {
+      "flags": "",
+      "unsupported": "",
+      "git": "https://github.com/solidiquis/erdtree",
+      "bins": [
+        "et"
+      ]
+    },
+
+    "cargo-semver-checks": {
+      "flags": "",
+      "unsupported": "",
+      "git": "https://github.com/obi1kenobi/cargo-semver-checks",
+      "bins": [
+        "cargo-semver-checks"
+      ]
     }
   },
-  "allowlist": "wasm-pack,cargo-update"
+  "allowlist": "erdtree,cargo-semver-checks"
 }

--- a/crates.json
+++ b/crates.json
@@ -505,12 +505,12 @@
 
     "cargo-semver-checks": {
       "flags": "",
-      "unsupported": "",
+      "unsupported": "x86_64-unknown-netbsd,x86_64-unknown-illumos,x86_64-sun-solaris,i686-unknown-freebsd",
       "git": "https://github.com/obi1kenobi/cargo-semver-checks",
       "bins": [
         "cargo-semver-checks"
       ]
     }
   },
-  "allowlist": "erdtree,cargo-semver-checks"
+  "allowlist": "cargo-semver-checks"
 }

--- a/crates.json
+++ b/crates.json
@@ -494,5 +494,5 @@
       ]
     }
   },
-  "allowlist": "cargo-zigbuild"
+  "allowlist": "wasm-pack,cargo-update"
 }

